kura/org.eclipse.kura.core.net/META-INF/MANIFEST.MF

```diff
@@ -12,6 +12,7 @@ Import-Package: org.eclipse.kura;version="[1.0,2.0)",
  org.eclipse.kura.core.util;version="[1.0,2.0)",
  org.eclipse.kura.executor;version="[1.0,2.0)",
  org.eclipse.kura.net;version="[2.0,3.0)",
+ org.eclipse.kura.net.configuration;version="[1.0,2.0)",
```

Comment:
Are we compliant with the 1.0 API or do we need to make it strictly requiring the current net.configuration version?

Reply:
Mhhh... the metatype returned by the `NetworkConfigurationServiceCommon` class with the `getDefinition` method has the same properties and type as before. The only things changed are the default values: before the defaut values were always an empty string, while now they are different values. So, the old code is compatible, but the behavior is a bit different (even if the default values added by the networking classes are the same). Probably it's better to increase the version of the `org.eclipse.kura.net.configuration` to 1.1. What do you think?

Reply:
Agreed, better to stay safe